### ISSUE TITLE
Error in setup

### ISSUE THREAD

**Issue by hmorkemo** [iMicknl/ha-nest-protect#166]
### The problem

After upgrading to the latest version (0.3.11) I get this error when I restart HA or reload nest protect:
```
2023-02-21 20:34:36.077 DEBUG (MainThread) [custom_components.nest_protect] Migrating from version 3
2023-02-21 20:34:36.077 DEBUG (MainThread) [custom_components.nest_protect] Migration to version 3 successful
2023-02-21 20:34:36.077 ERROR (MainThread) [homeassistant.config_entries] Error setting up entry Nest Protect for nest_protect
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 382, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/nest_protect/__init__.py", line 55, in async_setup_entry
    refresh_token = entry.data[CONF_REFRESH_TOKEN]
KeyError: 'refresh_token'

```

And all the devices (3) are unavailable
It worked yesterday, in 0.3.10.

### What version of this integration (ha-nest-protect) has the issue?

0.3.11

### What version of Home Assistant Core has the issue?

2023.2.5

### Device / Model

Model: Topaz-2.33, Firmware: 3.4.1rc1

### Home Assistant log

<details><summary>Logs</summary>
```
Copy/paste any log here, between the starting and ending backticks (`)
```

</details>

**Comment by iMicknl** [comment]
Did you use the cookie version before? If so, you shouldn't have updated since the update does not support cookie auth yet. 

Best is to revert to the previous version.

**Comment by hmorkemo** [comment]
I tried installing older versions all the way back to 0.3.8 (which is from before my installation) using HACS.
The same happens in .8 .9 and .10 versions.

**Comment by iMicknl** [comment]
@hmorkemo did you use the cookie auth method before? If so, you didn't install my version, but another version...

**Comment by hmorkemo** [comment]
I'm confused.
I have now deleted the next protect integration.
When I try to add it again, it appears twice in the repository list.
I'll try the the top one (with 189 likes).
I can't get past the invalid_request error, even if I follow the advice in #105.
When I select the other one, it says that the repository is owned by iMicknl. When I try to go to Github, I arrive at nicoinch/ha-nest-protect, whici is forked from the iMickNL one.
I tried the nicoinch one as well. There I get 
`Translation Error: The intl string context variable "url" was not provided to the string "To link your Google account, authorize your account.
`
when I try to add the integration. So no luck there either.
Something is wrong, or at least hard to understand.
